prdoc/pr_5443.prdoc

```diff
@@ -0,0 +1,10 @@
+crates:
+- name: frame-remote-externalities
+  validate: false
```

Comment:
```suggestion
  bump: patch
```
Validate false is like not adding it at all.

Reply:
ah I see, that makes total sense. Though didn't expect the url to be hard coded in the runtime code.
Let me commit this and see whether I'd be eventually able to merge this PR. :)
Thanks for taking a look.